### PR TITLE
[MSRCH-2569] Added filtering examples to Live Search product-search topic

### PR DESCRIPTION
**Pull request by dobooth**
## Description

DO NOT MERGE
Workis in  https://github.com/AdobeDocs/commerce-webapi/pull/102 instead.
This PR adds examples around filtering on facets in the Live Search productSearch query.

### CODE REVIEW

**Review by keharper — COMMENTED** [review]
As I mentioned in a Slack message, there's new undocumented schema. You'll need to add docs for `CategoryView` and `CategoryViewInterface`, at minimum.

File: src/pages/graphql/schema/live-search/queries/product-search.md [hunk 1]
```diff
@@ -259,13 +259,164 @@ The Catalog Service [products query](../catalog-service/products.md) describes t
...
+    },
+    {
+        "id": "35",
+        "title": "collections/performance-fabrics",
+        "count": 5
+    }
+]
+```
+
+Example 2: A category browse page when filtering on a category.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply src/pages/graphql/schema/live-search/queries/product-search.md:379]: Done

File: src/pages/graphql/schema/live-search/queries/product-search.md [hunk 2]
```diff
@@ -776,6 +1759,24 @@ Field | Data Type | Description
 `attribute` | String! | The attribute code of a product attribute
 `direction` | SortEnum! | ASC (ascending) or DESC (descending)
 
+### QueryContextInput data type {#QueryContextInput}
+
+The `QueryContextInput` object can contain the following fields.
+
+Field | Data Type | Description
+--- | --- | ---
+`customerGroup` | String! | Customer group code. For storefront clients, this value is available in the `dataservices_customer_group` cookie.
```
Comment (keharper) [inline src/pages/graphql/schema/live-search/queries/product-search.md:1768]:
```suggestion
`customerGroup` | String! | The customer group code. For storefront clients, this value is available in the `dataservices_customer_group` cookie.
```

File: src/pages/graphql/schema/live-search/queries/product-search.md [hunk 3]
```diff
@@ -259,26 +295,972 @@ The Catalog Service [products query](../catalog-service/products.md) describes t
...
+    ]
+  }
+}
+```
+
+</details>
+
+Example 2: A "category browse" page when filtering on a category.
+
+The user navigates to "Womens -> Bottoms" and filters on "pants". In this case, both "Pants" and "Shorts" appear as facets in the layered navigation.
```
Comment (keharper) [inline src/pages/graphql/schema/live-search/queries/product-search.md:856]:
```suggestion
The shopper navigates to "Womens -> Bottoms" and filters on "pants". In this case, both "Pants" and "Shorts" appear as facets in the layered navigation.
```

**Review by keharper — COMMENTED** [review]
This topic needs to be reorganized. The category filters are too different.

1. Remove the Field Reference heading and bump its subheads up a level. You may need to rethink the content of the material that immediately precedes Field Reference.
2. Add two subsections to `filter`: Product attribute filters and category filters. The topic currently describes product attribute filters, and the attributes being filtered can be any pre-defined product attributes or custom attributes. The category filters appear to be hardcoded values.  The catalog filters should describe only the basics of defining the filter and have very small code samples.
3. The examples each get their own heading. The example currently titled "Live Search" will need to be retitled to be less generic.

File: src/pages/graphql/schema/live-search/queries/product-search.md [hunk 4]
```diff
@@ -259,26 +295,972 @@ The Catalog Service [products query](../catalog-service/products.md) describes t
...
+This example shows how to filter returned facets when browsing a category page.
+
+`categoryPath` performs strict filtering, meaning that the facets returned are limited to the immediate children of the current category page.
+
+If you select "Gear -> Fitness Equipment" from the category navigation, the filter would look like:
+
+```graphql
+query productSearch {
+  productSearch(
+    phrase: "",
```
[the remark this answers is not in the document]
Comment (dobooth) [reply src/pages/graphql/schema/live-search/queries/product-search.md:322]: Addedd at 309 and in the appropriate description and attribute table.

**Comment by dobooth** [comment]
Closing as this branch got into a strange state when rebasing. Not willing to risk any release trouble so moving to https://github.com/AdobeDocs/commerce-webapi/pull/102

**Pull request closed without merging**